warp/stats.go

```diff
@@ -0,0 +1,36 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package warp
+
+import (
+	"github.com/ava-labs/subnet-evm/metrics"
+)
+
+type verifierStats struct {
+	messageParseFail metrics.Counter
```

Comment:
should we have some success metrics as well? (or are those available through the p2p handler?)